examples/create-react-app/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@ckb-lumos/lumos": "workspace:^",
```

Comment:
please switch to 'canary' to enable the example to work on the CodeSandbox

Reply:
Updated.